### PR TITLE
store: set retry flag for release 5.3 

### PR DESCRIPTION
**Pull request by ekexium**
### What problem does this PR solve?

Issue Number: ref #34875

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by sticnarf** [comment]
@ekexium Please fix the compilation failures.

**Comment by ekexium** [comment]
> @ekexium Please fix the compilation failures.

Awaiting https://github.com/pingcap/tidb/pull/30912

**Comment by ekexium** [comment]
PTAL @sticnarf @cfzjywxk

File: go.mod [hunk 1]
```diff
@@ -65,7 +65,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2
-	github.com/tikv/client-go/v2 v2.0.0-alpha.0.20211221095044-38d30d5632a8
+	github.com/tikv/client-go/v2 v2.0.0-alpha.0.20220601055058-46205016d898
```
Comment (cfzjywxk) [inline go.mod:68]: There's a new pick in the client-go v5.3 branch. https://github.com/tikv/client-go/commits/tidb-5.3

**Comment by ekexium** [comment]
It will be fixed in https://github.com/pingcap/tidb/pull/35398

**Pull request closed without merging**